source/Glossary/throttling.md

```diff
@@ -1,37 +1,31 @@
+  description: Learn more about deferrals and throttling
+  keywords: deferred, deferral, throttling, bounce, block
+title: Throttling
+weight: 0
+layout: page
 navigation:
   show: false
 ---
 
-So, you tried to send a bunch of email, and when you checked your [Event webhook]({{root_url}}/API_Reference/Webhooks/event.html) or your [Email Activity]({{root_url}}/User_Guide/Delivery_Metrics/email_activity.html) page, you see a number of messages are marked as "Deferred" instead of "Delivered". Or maybe you're looking at your [Email Reports]({{root_url}}/User_Guide/Delivery_Metrics/email_reports.html), and noticing a lot of addresses in your [Blocks]({{site.app_url}}/suppressions/blocks), but not many in your Hard Bounces. What? Why aren't your recipients currently in awe at the glory of your amazing message?
+So, you tried to send a bunch of email, and when you checked your [Event webhook]({{root_url}}/API_Reference/Webhooks/event.html) or your [Email Activity]({{root_url}}/User_Guide/Delivery_Metrics/email_activity.html) page, you see a number of messages are marked as "Deferred" instead of "Delivered". Or maybe you're looking at your [Email Reports]({{root_url}}/User_Guide/Delivery_Metrics/email_reports.html), and noticing a lot of addresses in your [Blocks]({{site.app_url}}/suppressions/blocks), but not many in your Bounces. What? Why aren't your recipients currently in awe at the glory of your amazing message?
 
-First of all, rest assured that our system is designed with throttling (also known as "deferral" these terms mean the same thing in the email world, and may be used interchangeably) in mind. We continue to attempt to send messages for up to 72 hours until the message is delivered. Normally, it takes between a few minutes to a few hours until the message is delivered.
+Sendgrid systems have been designed with throttling in mind. We will continue to attempt to send messages for up to 72 hours until the message is delivered. Normally, it takes between a few minutes to a few hours until a deferred message is delivered, it's all up to the receiving mail server.
 
-But what is throttling? Well, it's when a receiving server refuses a message, but with a specific explanation that tells our server it's only a temporary refusal. They usually say things like "user is receiving mail at too great a rate right now, please try again later", or "user's mailbox is over quota." One important thing to note is that these errors are almost always a **400** class of error.
+Throttling is when a receiving server refuses a message, but with a specific explanation that tells our server it's only a temporary refusal. They will usually say things like "user is receiving mail at too great a rate right now, please try again later", or "user's mailbox is over quota." An important thing to note is that these errors are almost always a **400** sytle error, indicating a temporary failure. Unlike a **500** style, which indicates a permanent failure. 
```

Comment:
~~An important thing to note is that these errors are almost always a 400 sytle error, indicating a temporary failure. Unlike a 500 style, which indicates a permanent failure.~~

These errors are most often **400** errors - indicating a temporary failure. They are not often **500** errors - indicating a permanent failure.